src/wallet/storage.rs

```diff
@@ -16,8 +17,33 @@ use super::{error::WalletError, fidelity::FidelityBond};
 
 use super::swapcoin::{IncomingSwapCoin, OutgoingSwapCoin};
 
+#[derive(Debug, Eq, PartialEq, Deserialize, Serialize)]
+pub enum ListUnspentInfo {
+    ListUnspentResultEntry(ListUnspentResultEntry),
```

Comment:
Possible to reduce `ListUnspentResultEntry` to only information we need to store? No need to just copy the whole core-rpc struct.